src/Bridge/Symfony/XliffFileDumper.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Translation\PlatformAdapter\PhraseApp\Bridge\Symfony;
+
+use Symfony\Component\Translation\Dumper\FileDumper;
+use Symfony\Component\Translation\MessageCatalogue;
+
+/**
+ * Custom XliffFileDumper for use with phrase app
+ */
+class XliffFileDumper extends FileDumper
```

Comment:
@Nyholm @prolic since the id attribute created in the symfony dumper is useless. It might be a chance to have it changed there? Maybe something like this:
- lowercase everything
- replace every "." and " " with _ 
- remove all special chars
- shorten the string to (maybe) 100 chars
- append a smaller unique id (maybe just substr of the md5 currently created)

In an environment where the source is a phrase instead of a key, this will work too and create a more useful id. I am not sure if this might be considered as a BC break - but it shouldn't because you can't reverse the md5 hash anyway.